randovania/gui/main_window.py

```diff
@@ -190,18 +204,44 @@ def __init__(
             logo.clicked.connect(partial(self._select_game, game))
             logo.setVisible(game.data.development_state.can_view())
 
-            def highlight_logo(label: ClickableLabel, active: bool):
+            logo.on_hover_effect = QtWidgets.QGraphicsColorizeEffect()
+            logo.on_hover_effect.setStrength(0.5)
+            logo.setGraphicsEffect(logo.on_hover_effect)
+            logo.on_hover_effect.setEnabled(False)
+
+            multibanner = QtWidgets.QLabel(pack_tile)
+            multibanner.setPixmap(QtGui.QPixmap(os.fspath(banner_img_path)))
+            multibanner.setScaledContents(True)
+            multibanner.setFixedSize(bannerSize, bannerSize)
+            multibanner.setVisible(game.data.defaults_available_in_game_sessions)
+
+            multibanner.color_effect = QtWidgets.QGraphicsColorizeEffect()
+            multibanner.color_effect.setStrength(0.5)
+            multibanner.color_effect.setColor(QtGui.QColor(70, 200, 80))
+            multibanner.setGraphicsEffect(multibanner.color_effect)
+            multibanner.move(0, 2)
+
+            multiicon = QtWidgets.QLabel(pack_tile)
+            multiicon.setPixmap(QtGui.QPixmap(os.fspath(multiworld_img_path)))
+            multiicon.setScaledContents(True)
+            multiicon.setFixedSize(bannerSize / 2, bannerSize / 2)
+            multiicon.setVisible(game.data.defaults_available_in_game_sessions)
+            multiicon.move(bannerSize / 4, 2 + bannerSize / 4)
+            if multiicon.isVisible:
```

Comment:
Isn't this a function?

Reply:
I'm not sure I understand. Is what a function?

Reply:
`isVisible`

Reply:
According to qt documentation at least, it's shown as a function
https://doc.qt.io/qt-5/qwidget.html#visible-prop

Reply:
Since that change may affect behaviors, I will hold off until I get home with a computer.